com.civicrm.hrjobroles/js/src/job-roles/controllers/modal-dialog.js

```diff
@@ -0,0 +1,23 @@
+define([
+  'job-roles/controllers/controllers'
+], function (controllers) {
+    'use strict';
+
+    controllers.controller('ModalDialogCtrl',['$scope','$uibModalInstance', '$timeout', 'content', '$log',
+        function ($scope, $modalInstance, $timeout, content, $log) {
+            $log.debug('Controller: ModalDialogCtrl');
+
+            $scope.title = content.title || 'CiviHR Job Roles';
+            $scope.msg = content.msg || '';
+            $scope.copyConfirm = content.copyConfirm || 'Yes';
+            $scope.copyCancel = content.copyCancel || 'Cancel';
+
+            $scope.confirm = function (action) {
+                $modalInstance.close(action || true);
```

Comment:
Do you mean `$modalInstance.close(action || true);` should completely depend on action?
i.e `$modalInstance.close(action);` ?